samples/test/backups.test.js

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {assert} = require('chai');
+const {describe, it, before, after} = require('mocha');
+const uuid = require('uuid');
+const {execSync} = require('child_process');
+const {Bigtable} = require('@google-cloud/bigtable');
+
+const exec = cmd => execSync(cmd, {encoding: 'utf8'});
+
+const bigtable = new Bigtable();
+
+const instanceId = `gcloud-tests-${uuid.v4()}`.substr(0, 30); // Bigtable naming rules
+const clusterId = `gcloud-tests-${uuid.v4()}`.substr(0, 30); // Bigtable naming rules
+const tableId = `gcloud-tests-${uuid.v4()}`.substr(0, 30); // Bigtable naming rules
+const backupId = `gcloud-tests-${uuid.v4()}`.substr(0, 30); // Bigtable naming rules
+const instance = bigtable.instance(instanceId);
+
+describe('backups', () => {
+  before(async () => {
+    await instance.create({
```

Comment:
I seem to remember there being a fairly low cap on the number of clusters you can have at a time.  I created `samples/test/util.js` to help do a check-in/out re-use thing to keep the number of clusters we create down if possible.